openhands/llm/llm.py

```diff
@@ -351,6 +347,9 @@ async def check_stopped():
         )
         async def async_acompletion_stream_wrapper(*args, **kwargs):
             """Async wrapper for the litellm acompletion with streaming function."""
+            # Merge async_completion_kwargs with the provided kwargs
+            merged_kwargs = {**async_completion_kwargs, **kwargs}
```

Comment:
Why do we merge here kwargs, when in async we didn't need to? Sorry, it's just not immediately clear to me.

Reply:
Removed that now.